### PR TITLE
Modified topic list.

### PR DESCRIPTION
**Pull request by m-goblet**
Now every user can see all topics and their manager, except the confidential topics.

### CODE REVIEW

File: samples/templates/samples/list_topics.html [hunk 1]
```diff
@@ -25,14 +25,25 @@
...
 {% block frame_content %}
 {% if topics %}
-  <ul>
-    {% for topic in topics %}
+  <table class="padding5">
+  <tbody>
+    <tr>
+      <td><b>{% trans 'topic'|capfirst %}</b></td>
+      <td><b>{% trans 'topic manager'|capfirst %}</b></td>
+    </tr>
```
Comment (bronger) [inline samples/templates/samples/list_topics.html]: Please use <th> instead of <td><b> here.

File: samples/templates/samples/list_topics.html [hunk 2]
```diff
@@ -25,14 +25,25 @@
...
+    {% for topic,manager,edit_able in topics %}
```
Comment (bronger) [inline samples/templates/samples/list_topics.html]: "editable" should be one word.

File: samples/views/topic.py [hunk 3]
```diff
@@ -164,11 +164,17 @@ def list_(request):
...
-        raise Http404("Can't find any topics that you can edit.")
-    return render(request, "samples/list_topics.html", {"title": _("List of all topics"), "topics": editable_topics})
+    topics = []
+    for topic in Topic.objects.filter(parent_topic=None).iterator():
+        if topic.confidential and user not in topic.members.all() and not user.is_superuser:
+            continue
+        edit_able = False
+        if permissions.has_permission_to_edit_topic(user, topic):
+            edit_able = True
+        topics.append((topic, topic.manager, edit_able))
```
Comment (bronger) [inline samples/views/topic.py]: You needn't pass `topic.manager` to the template.  The template can access the manager directly.

**Comment by bronger** [comment]
The general idea looks good to me.  The whole topic thing should be re-thought sometime, but in the meantime, the PR is a good move.

**Comment by m-goblet** [comment]
I made the changes you suggested.

**Pull request merged**